src/directConnectManager.ts

```diff
@@ -204,7 +204,7 @@ export class DirectConnectionManager {
         : await tryToCreateConnection(spec, dryRun);
       const connectionId = connection.spec.id as ConnectionId;
       if (!dryRun) {
-        await window.withProgress(
+        window.withProgress(
```

Comment:
Removing this means `catch` block won't be triggered for the async exceptions here.

Reply:
We shouldn't need to specifically catch anything around that progress notification or `waitForConnectionToBeUsable()`, just the `tryToCreate`/`tryToUpdate` calls. (https://github.com/confluentinc/vscode/pull/798 changes `waitForConnectionToBeUsable()` to return `null` instead of throwing)